package.json

```diff
@@ -788,7 +854,6 @@
     "@types/request-promise-native": "^1.0.15",
     "@types/semver": "^5.5.0",
     "adm-zip": "^0.4.11",
-    "azure-storage": "^2.8.1",
```

Comment:
Is there any reason to move azure-storage from devDependencies to dependencies?

Reply:
devDependencies aren't packaged into the VSIX, they're only for building and testing. Now that you're using it at runtime it needs to be in dependencies.